prime_factor/prime_factor.py

```diff
@@ -0,0 +1,30 @@
+import math
+
+
+def prime_factor(num):
+    """
+    Finds all the prime factor of a number.
+
+    :param num : number whose prime factors we want to find.
+    :return : return a list of prime factors of number.
+    """
+    factor_list = []
+    sqrt_num = int(math.sqrt(num)) + 1
+    for start in range(2, sqrt_num):
+        if num % start == 0:
+            factor_list.append(start)
+            while (num % start == 0):
+                num /= start
```

Comment:
This implementation might fail for numbers like 35. Please follow https://github.com/iiitv/algos/blob/master/prime_factor/primeFactor.js . More specifically 

```js
if (n !== 1) {
	primes.push(n);
}
```